test/customer-test.js

```diff
@@ -39,4 +44,13 @@ describe('customer', () =>  {
   it('should have a password', () => {
     expect(customer.password).to.equal(`overlook2021`)
   })
+
+  it('should have a method to sum user\'s bookings', () => {
```

Comment:
Good catching adding another test. I know I can forget them sometimes if I add a property or method later on in the project.